infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
```

Comment:
So module called => `github_webhooks`?

What about the entrypoint? Should we use `__main__` as entry point? In the AWS example they called the entrypoint `lambda_handler`.

I personally don't mind. The reason why I called it index/handler as that was the default in AWS console and I though was some kind of non-written convention, but if you have something in mind let me know.

(But I agree with you that the module name is not the best, probably `github_webhooks` is better)